generators/spring-data-relational/templates/src/test/java/package/config/timezone/HibernateTimeZoneIT.java.ejs

```diff
@@ -65,16 +62,9 @@ class HibernateTimeZoneIT {
         dateTimeWrapper.setOffsetTime(OffsetTime.parse("14:30:00+02:00"));
         dateTimeWrapper.setLocalDate(LocalDate.parse("2016-09-10"));
 
-        dateTimeFormatter = DateTimeFormatter
-            .ofPattern("yyyy-MM-dd HH:mm:ss.S")
-            .withZone(ZoneId.of(zoneId));
-
-        timeFormatter = DateTimeFormatter
-            .ofPattern("HH:mm:ss")
-            .withZone(ZoneId.of(zoneId));
-
-        dateFormatter = DateTimeFormatter
-            .ofPattern("yyyy-MM-dd");
+        dateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.S");
+        timeFormatter = DateTimeFormatter.ofPattern("HH:mm:ss");
```

Comment:
```suggestion
        timeFormatter = DateTimeFormatter.ofPattern("HH:mm:ss").withZone(ZoneOffset.UTC);
```
Since the `timeFormatter` is constantly being used like that.